### ISSUE TITLE
Create a set-weather-update-interval command

### ISSUE THREAD

**Issue by scottbarnesg** [scottbarnesg/WeatherByLocation#24]
This command should allow the user to update the value of `minutes-between-updates` in `config.yml`.

This command should include a warning that this change will not take effect until the server is restarted or the plugin is reloaded.

**Comment by adrienzag** [comment]
Hello,

I would like to take on this issue but had a few clarifications i wanted to make:

**Naming**
Under the current config.yml file and within classes such as WeatherByLocation.java (line 113) it looks like we are referring to the minutes between updates as "minutes-between-updates" , should this be changed to "update-interval-minutes" or will we have two separate entries ?

**Warning**
The warning about the change not taking effect until server restart/plugin reload, should it show up after the fact or should it be almost like a prompt that will ask "XYZ will change and but will not take effect until reload of server etc ...do you want to proceed ?" (and the user is prompted with Y/N). The former makes more sense but I wanted to confirm before assuming.

**Interval Limits**
On a related note, currently the config.yml file recommends (for the key-value pair corresponding to "minutes-between-updates") a range of 10-100 minutes, should this be enforced or is this only a suggestion for the user ?

Please let me know if there is anything I am missing.

Thanks

**Comment by scottbarnesg** [comment]
Hi @adrien2-public! Great questions. Here are my thoughts:

- Naming: I mistyped this in the issue description, it  should be `minutes-between-updates`. I've fixed the issue description - good catch.
- Warning: The way I've handled warnings/errors is sending the player who runs the command a message, [like this](https://github.com/scottbarnesg/WeatherByLocation/blob/ce4eab85adb42a162d3a5bf61020ac2bd145ea4c/src/main/java/mc/spigot/weatherbylocation/SetLocationCommand.java#L24), and log the warning in the server console. I think we can just send them a message with the warning without the `y/n` prompt. 
- Interval Limits: The hard requirement for the interval is that its a positive integer. The suggestions are a "reasonable" range so that we don't overload the weather API. Lets enforce that the value must be `>=1`, but otherwise let the user choose.

Let me know if you have any more questions!
